python/hsfs/feature_group.py

```diff
@@ -1733,7 +1734,54 @@ def read_changes(
             .read(False, "default", read_options)
         )
 
-    def find_neighbors(self, embedding, col=None, k=10, filter=None, min_score=0):
+    def find_neighbors(
+        self,
+        embedding: List[Union[int, float]],
+        col: Optional[str] = None,
```

Comment:
Why is it "col" here?
Earlier, we have "features" as a parameter (for Embedding objects). 
'col' is clear to what it means - i just want a consistent API.

Reply:
Because in `EmbeddingIndex` it is the `EmbeddingFeature`. But here user just need to provide the feature name as a string, in `feature_view.find_neighbors` the argument is also called `feature` because it require a feature object. so I use a different name here so that users won't confuse about the type.